packages/wordcount/src/defaultSettings.js

```diff
@@ -50,7 +50,7 @@ export const defaultSettings = {
 			'[',
 
 			// Basic Latin (extract)
-			'\u0021-\u0040\u005B-\u0060\u007B-\u007E',
+			'\u0021-\u002F\u005B-\u0060\u007B-\u007E',
```

Comment:
It looks like you removed extra range \u003A-\u0040 which are not numbers

Reply:
https://jrgraphix.net/r/Unicode/0020-007F

Reply:
Yes, this is the main reason why numbers are not count as words. Because they are removed before the real count logic get a hold of them.

The purpose of this PR is about changing the behaviour of the word count logic because it behaves differently from the most other word count functions in other software/lib etc. The ticket #26339 details it.

Reply:
I mean numbers are \u0030-\u0039, not \u0030-\u0040, right?

Reply:
Yes, you are right! I should update that.